### DISCUSSION TITLE
pass the schema when you instantiate the client.

### DISCUSSION THREAD

**Discussion by arismarko** [prisma/prisma#15294] [Q&A]
Hello!

Is there any way we could pass the schema as a raw text?
```ts
  const prisma = new PrismaClient( 
   {datasources: {
      db: {
        url: 'URL_FROM_PROXY',
      },
    }
  });

  // prisma.inlineSchema = require("raw-loader!../prisma/schema.prisma");
  console.log(prisma);

  const stores = await prisma.store.findMany();
```

Can you see the line I commented it, can i require it and pass it as a raw text somehow?

This is an experiment when i run this by itself it works but when I run from another client i am getting an error (it is running serverside from another resource)

such as this 

![Screenshot 2022-09-13 at 14 12 56](https://user-images.githubusercontent.com/281849/189910733-4fc9a566-542e-424e-9964-552413c7d966.png)

**Comment by austincrim** [comment]
Hey @arismarko 👋,

This kind of workflow is not supported by the Prisma Client. Please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if this is something you would like to see added.

Could you explain your use case a little more as well?

**Comment by arismarko** [comment]
Basically I am doing an experiment for a talk with microfrontends 

There is a container app that runs the nextjs app serverside I can see that the app is trying to make a call to the prisma proxy but it fails 


These are the logs:

```
[1] prisma:info Calling https://aws-us-east-1.prisma-data.com/4.3.1/96632077410d3e3d43bc1b2012565023cd6edd3fb8c219e9351ba3b50878c169/graphql (n=1)
[1] prisma:error Error while querying: Cannot fetch data from service:
[1] Rm is not a function
[1] prisma:warn This request can be retried
[1] prisma:warn Retrying after 62ms
[1] prisma:info Calling https://aws-us-east-1.prisma-data.com/4.3.1/96632077410d3e3d43bc1b2012565023cd6edd3fb8c219e9351ba3b50878c169/graphql (n=2)
[1] prisma:error Error while querying: Cannot fetch data from service:
```

When I run the app it self it works and it returns the data and all that I am not sure why the call to the aws is blocked.

**Reply by pantharshit00** [reply]
Hey @arismarko,

It would be great if you can open a feature request for this: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

That will ensure that this is properly tracked and a decision will be made whether we want to implement this or not.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
